domain/schema/model/sql/0022-resource.sql

```diff
@@ -133,3 +134,13 @@ CREATE TABLE unit_resource (
     REFERENCES unit (uuid),
     PRIMARY KEY (resource_uuid, unit_uuid)
 );
+
+CREATE TABLE resource_oci_image_metadata_store (
+    resource_uuid TEXT NOT NULL,
+    registry_path TEXT NOT NULL,
+    username TEXT,
+    password TEXT,
```

Comment:
Considering this is plaintext password and a password that we don't have ownership of, should we at least add a note that this should either be encoded with base64 or use a [secretbox](https://pkg.go.dev/golang.org/x/crypto/nacl/secretbox). Then we can seal the box with some sort of key, potentially the admin password?

Reply:
I changed it to a password hash and salt like we do user passwords.